### DISCUSSION TITLE
Is babel 7 compatible with webpack 3?

### DISCUSSION THREAD

**Discussion by imteekay** [babel/babel#13613] [Q&A]
I'm currently working on a project that is using webpack 3 and babel 6. I was wondering if it's possible to bump babel to version 7 without upgrading webpack or should I bump them together to 4/5 (webpack) and 7 (babel)?

**Comment by JLHwung** [comment]
Babel 6 is not maintained anymore. Consider update both webpack, babel and babel-loader to latest versions. See https://babel.dev/docs/en/v7-migration for migration guide to Babel 7.

**Reply by imteekay** [reply]
But is it possible to upgrade babel to version 7 while using webpack 3? It would be nice to migrate them in small steps.

**Reply by JLHwung** [reply]
You can try https://github.com/babel/babel-loader/tree/7.x . But Babel-loader v7 is not actively maintained.
